qpc/server/__init__.py

```diff
@@ -15,6 +15,8 @@
 CONFIG = 'config'
 LOGIN = 'login'
 LOGOUT = 'logout'
+STATUS = 'status'
```

Comment:
Copyright should be updated to have 2017-2018 date.

Reply:
fixed 👍